lbrynet/wallet/manager.py

```diff
@@ -15,6 +15,12 @@
 log = logging.getLogger(__name__)
 
 
+class ReservedPoints:
+    def __init__(self, identifier, amount):
+        self.identifier = identifier
+        self.amount = amount
+
+
```

Comment:
Why did this get moved? Actually, it would be even better if it was deleted altogether, could you do the necessary refactoring to make that happen? I don't think it's used in many places.

Reply:
Yes, please remove it and make the necessary changes.

Reply:
Ok, will/might take some time. If I don't push a commit for this today, then don't treat this as a blocker. But rest assured this is on top of my list.